synapseclient/client.py

```diff
@@ -1046,34 +1044,44 @@ def _createAccessRequirementIfNone(self, entity):
         if len(existingRestrictions['results']) <= 0:
             self.restPOST('/entity/%s/lockAccessRequirement' % id_of(entity), body="")
 
-    def _getEntityBundle(self, entity, version=None, bitFlags=0x800 | 0x40000 | 0x2 | 0x1):
+    def _getEntityBundle(self, entity, version=None, requestedObjects=None):
         """
         Gets some information about the Entity.
 
         :parameter entity:      a Synapse Entity or Synapse ID
         :parameter version:     the entity's version (defaults to None meaning most recent version)
-        :parameter bitFlags:    Bit flags representing which entity components to return
-
-        EntityBundle bit-flags (see the Java class org.sagebionetworks.repo.model.EntityBundle)::
-
-            ENTITY                     = 0x1
-            ANNOTATIONS                = 0x2
-            PERMISSIONS                = 0x4
-            ENTITY_PATH                = 0x8
-            HAS_CHILDREN               = 0x20
-            ACL                        = 0x40
-            FILE_HANDLES               = 0x800
-            TABLE_DATA                 = 0x1000
-            ROOT_WIKI_ID               = 0x2000
-            BENEFACTOR_ACL             = 0x4000
-            DOI                        = 0x8000
-            FILE_NAME                  = 0x10000
-            THREAD_COUNT               = 0x20000
-            RESTRICTION_INFORMATION    = 0x40000
+        :parameter requestedObjects:    A dict indicating settings for what to include
 
-        For example, we might ask for an entity bundle containing file handles, annotations, and properties::
+        default value for requestedObjects is::
+
+            requestedObjects = {'includeEntity': True,
```

Comment:
Are we likely to pass False for any of these values (i.e. does the backend API itself have defaults for some of these values)? If not then a set of keys may be simpler than a dictionary of True values.

Reply:
document that unneeded keys (set to False) can be omitted.